site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx

```diff
@@ -92,7 +92,7 @@ const CreateSocialMediaPost = (props: CreateSocialMediaPostPageProps): ReactElem
                                     stateUpdater(props.disruptionSummary, "messageContent");
```

Comment:
For social media posts, we are copying over the summary not the description - is this intended behaviour as ticket says to copy over disruption description?

If we do need to copy over the disruption description then we may face issues with character length. A disruption description can be 1000 characters whereas a social media post can be 200 characters.

Can we get some clarification from Chris?

Reply:
he said to increase

Reply:
done

Reply:
Social media posts are still copying the disruption summary not the disruption description.

Also I'm not 100% we can increase the limit to 1000 characters for social media posts. Twitter's character limit is 280 - I'm not sure what happens if we exceed this limit.